website/content/docs/commands/plugin/register.mdx

```diff
@@ -24,17 +24,22 @@ Success! Registered plugin: my-custom-plugin
 
 Register an Enterprise plugin:
 
--> **Note:** See [Plugin managment for Enterprise plugins](/vault/docs/plugins/plugin-management#enterprise-plugins)
+<Note>
+
+Refer to the [Enterprise plugins](/vault/docs/plugins/plugin-management#enterprise-plugins) section
+of the Plugin management page for Vault compatibility requirements.
+
+</Note>
```

Comment:
I think something weird happened with the suggestion I added. We should delete line 33, it's left over from the previous note somehow.